src/common/featuremanager/FeatureManagerService.js

```diff
@@ -956,6 +956,18 @@
       }
       return deferredResponse.promise;
     };
+
+    this.coordinatesWithinRange = function(coords1, coords2, range) {
+      if (coords1.length != coords2.length) {
+        return false;
+      }
+      for (var i = 0; i < coords1.length; i++) {
+        if (Math.abs(coords1[i] - coords2[i]) > range) {
+          return false;
+        }
+      }
+      return true;
+    };
```

Comment:
It's okay to leave this here, it actually makes it easier to test the function since it's "exported."